### ISSUE TITLE
Regarding the issue of errors when running tools/relation_train_net.py during the training phase

### ISSUE THREAD

**Issue by Young-Loser** [Maelic/SGG-Benchmark#33]
First of all, thank you very much for your excellent work! I followed your documentation to train my own SGDet model, but I keep encountering the following error:
<img width="773" alt="1728648985610" src="https://github.com/user-attachments/assets/cf1b14fd-a8bf-4603-ad15-304090ad453f">
It seems that some packages have not been imported.However, I can confirm that all the required packages have been downloaded, imported, and the environment has been activated according to your instructions.In response to this, I also conducted a simple test, and the code runs normally.
<img width="254" alt="1728649121273" src="https://github.com/user-attachments/assets/5dbce441-f26b-4f69-842f-e2fed90ed6c6">
I consulted some people in the lab, but no one knows how to solve this issue. It seems like there might be a problem with the code functions or something else. I really hope to get your response! Thank you very much!!!
Here is the command I used to run the script for training:`CUDA_VISIBLE_DEVICES=1,2 torchrun --master_port 10025 --nproc_per_node=2 tools/relation_train_net.py --task sgdet --save best --config-file "/Dpan/wyc/realtime_rwsg/SGG-Benchmark/configs/VG150/e2e_relation_yolov8m.yaml" MODEL.ROI_RELATION_HEAD.PREDICTOR MotifPredictor SOLVER.IMS_PER_BATCH 12 TEST.IMS_PER_BATCH 2 DTYPE "float16" SOLVER.MAX_EPOCH 20 MODEL.PRETRAINED_DETECTOR_CKPT /Dpan/wyc/realtime_rwsg/PREMODEL/yolov8m_vg150.pt OUTPUT_DIR /Dpan/wyc/realtime_rwsg/MODELSAVE`

**Comment by Maelic** [comment]
Hi, you can just do ```pip install wandb``` or just remove this line [https://github.com/Maelic/SGG-Benchmark/blob/c0e840c1e7ce1488c19161cb1ea1c8992f6572e8/tools/relation_train_net.py#L10](https://github.com/Maelic/SGG-Benchmark/blob/c0e840c1e7ce1488c19161cb1ea1c8992f6572e8/tools/relation_train_net.py#L10) 

Wandb is a tool for visualizing your training, which is very convenient in my opinion, if you want to use it you can have a look at [https://wandb.ai/](https://wandb.ai/)

**Comment by Maelic** [comment]
Just do ```pip install yacs``` 🤗

**Comment by Young-Loser** [comment]
> Just do `pip install yacs` 🤗

Thank you very much for your patient response! I eventually solved the issue, and it turned out that the problem was caused by modifications I made to the code. However, I’ve now encountered another strange issue. I would like to ask why the file informative_file does not have a path reference in the code? Moreover, the original code is commented out, and I haven’t been able to find where to download the informative_sg.json file.
<img width="359" alt="1728656477779" src="https://github.com/user-attachments/assets/797e6bda-7a20-49d6-bd2f-248658ceb714">
